packages/app-staking/src/Account/index.tsx

```diff
@@ -294,13 +299,14 @@ class Account extends React.PureComponent<Props, State> {
     }
 
     return (
+      isNominateOpen ?
```

Comment:
There _are_ actually very good reasons why some of these are done this way. 

In the cases where we do queries and display the info to the user, we want to fire them in the background, so there is no actual delay for the display of the info. It has the additional benefit that the subscriptions are not re-created each time as well.

(The latter probably only until we get multi support inside the API)